### PR TITLE
Add grafana json file for replication metrics

### PR DESCRIPTION
**Pull request by sinhaashish**
## Description
Add a new file names `minio-replication.json` which displays the replication metrics in Grafana dashboard.

## How to test this PR?

 Refer https://github.com/minio/minio/blob/master/docs/metrics/prometheus/grafana/README.md
 Use `minio-replication.json`  in the grafana dashboard to see the replication metrics 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

File: docs/metrics/prometheus/grafana/README.md [hunk 1]
```diff
@@ -14,3 +14,10 @@ Visualize MinIO metrics with our official Grafana dashboard available on the [Gr
 Refer to the dashboard [json file here](https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/minio-overview.json).
 
 ![Grafana](https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/grafana-minio.png)
+
+
+Replication metrics can also be viewed in grafana dashboard using [json file here](https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/minio-replication.json).
```
Comment (harshavardhana) [inline docs/metrics/prometheus/grafana/README.md]:
```suggestion
Replication metrics can also be viewed in the Grafana dashboard using [json file here](https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/minio-replication.json).
```

File: docs/metrics/prometheus/grafana/README.md [hunk 2]
```diff
@@ -14,3 +14,10 @@ Visualize MinIO metrics with our official Grafana dashboard available on the [Gr
...
+
+
+
+![Grafana](https://raw.githubusercontent.com/minio/minio/master/docs/metrics/prometheus/grafana/grafana-replication.png)
```
Comment (harshavardhana) [inline docs/metrics/prometheus/grafana/README.md:23]: The screenshot doesn't show any replication stuff - are you sure this is the correct screenshot?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

File: docs/metrics/prometheus/grafana/README.md (hunk 2 above)
Comment (sinhaashish) [reply docs/metrics/prometheus/grafana/README.md:23]: The screen shot has `Sent Bytes for Replication` and `Received Bytes for Replication` . The screenshot is large , You need to scroll. 

Screenshot from 2021-11-17 23-08-09

Screenshot from 2021-11-17 23-08-17

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by eco-minio — APPROVED** [review]
Tested, LGTM

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**